README.md

```diff
@@ -1 +1,223 @@
-# Wheels-4-Rent-frontend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
# Backend review

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] In reservations_controller.rb:

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/57421823/4244fc37-ddbb-4655-8684-4095fd076645)

What happens here if the reservation is not successfully destroyed?  KIndly, add some error handling, like you did in the create method (line 27) 👍